src/utils.js

```diff
@@ -114,13 +114,26 @@ function pluralize(str) {
 }
 exports.pluralize = pluralize;
 
+/**
+ * @deprecated use {@link injectReplacements} instead. This method has been removed in v7.
+ *
+ * @param {[string, ...unknown[]]} arr - first item is the SQL, following items are the positional replacements.
```

Comment:
@ephys I assume the issue is with this line
```
SyntaxError: Invalid regular expression: /[~][string, ..$/: Unterminated character class
```

Reply:
That's most likely it, and the docs check did not catch it

Reply:
PR ready https://github.com/sequelize/sequelize/pull/14513